### PR TITLE
Gui cosmetics

### PR DESCRIPTION
**Pull request by armengau**
- Color option for the main curves (coadded spectrum and noise, model)
- Improved widget layout when --no_vi_widgets is requested (which is the case of the interface to legacyviewer)
- Added a TextInput to select a spectrum

### CODE REVIEW

**Comment by armengau** [comment]
About the color option: eg. `plotspectra(colors=['black', 'blue', 'orange'])` 
or  `prospect_pages --colors black blue orange`
changes the colors of the coadded spectra, "model", and coadded noise vectors respectively.

- This makes it as simple as possible for the user to choose colors.
- No option to change the colors of single-arm spectra, as their color codes is not arbitrary but associated to blue, red and NIR (plus they're less used)

File: doc/changes.rst [hunk 1]
```diff
@@ -5,8 +5,12 @@ prospect's Change Log
 1.3.1 (unreleased)
 ------------------
 
+* Added a Text widget to enter a spectrum number by hand.
```
Comment (weaverba137) [inline doc/changes.rst:8]: Please add the PR number here.

**Review by weaverba137 — APPROVED** [review]
I don't see any issues with this PR. I just had a minor comment on the change log.

File: doc/changes.rst (hunk 1 above)
Comment (armengau) [reply doc/changes.rst]: done

**Pull request merged**